.github/workflows/weekly-tests.yaml

```diff
@@ -71,7 +71,11 @@ jobs:
                   ref: develop
             - name: Build VEGA_X86/gem5.opt
               run: scons build/VEGA_X86/gem5.opt -j`nproc`
-
+            - name: Get Pannotia inputs
+              working-directory: ${{ github.workspace }}/tests/gem5/gpu-pannotia-resources/pannotia-datasets
+              run: gsutil -m cp -r \ "gs://dist.gem5.org/dist/develop/datasets/pannotia/README.txt" \ "gs://dist.gem5.org/dist/develop/datasets/pannotia/bc"
+                  \ "gs://dist.gem5.org/dist/develop/datasets/pannotia/color" \ "gs://dist.gem5.org/dist/develop/datasets/pannotia/floydwarshall" \ "gs://dist.gem5.org/dist/develop/datasets/pannotia/mis"
+                  \ "gs://dist.gem5.org/dist/develop/datasets/pannotia/pagerank" \ "gs://dist.gem5.org/dist/develop/datasets/pannotia/sssp" \ .
```

Comment:
`gsutil` is mostly a tool for people to interact with their Google Cloud account. I think you need to associate a Google Account. Anyway, I doubt it's installed on our runners and it's really not the tool for the job. If we were to put the download here i'd just recommend `wget`, though I have a suggestion elsewhere in this review that we should put them in "tests_gpu_pannotia.py"